test/integration_test.rb

```diff
@@ -54,5 +54,20 @@ def test_selecting_test_even_though_addition_is_made_among_comments
         assert_match '1 runs, 1 assertions, 1 failures', output[:stdout]
       end
     end
+
+    class FizzBuzzMultiCode < TTNT::TestCase::FizzBuzzMultiCode
+      def test_code_files_option
+        fn = 'fizzbuzz.rb'
+        File.write(fn, File.read(fn).gsub(/"fizzbuzz"$/, "foo"))
+        output = rake('ttnt:test:run')
+        assert_match 'No test selected.', output[:stderr],
+          'Changing files which is not specified in code_files should not select tests.'
```

Comment:
Oops... thank you!
Fixed: https://github.com/Genki-S/ttnt/commit/77ae19197d8fe6c6da37aa45cdfe2990b1d1b1ad